common/app/model/meta.scala

```diff
@@ -59,12 +59,26 @@ object Fields {
       displayHint = apiContent.fields.flatMap(_.displayHint).getOrElse(""),
       isLive = apiContent.fields.flatMap(_.liveBloggingNow).getOrElse(false),
       sensitive = apiContent.fields.flatMap(_.sensitive),
-      shouldHideReaderRevenue = apiContent.fields.flatMap(_.shouldHideReaderRevenue),
+      shouldHideReaderRevenue = shouldHideReaderRevenue(apiContent),
       legallySensitive = apiContent.fields.flatMap(_.legallySensitive),
       firstPublicationDate = apiContent.fields.flatMap(_.firstPublicationDate).map(_.toJodaDateTime),
       lang = apiContent.fields.flatMap(_.lang)
     )
   }
+
+  private def shouldHideReaderRevenue(apiContent: contentapi.Content): Option[Boolean] = {
+    val cutoffDate = new DateTime("2017-07-03T12:00:00.000Z")
```

Comment:
@TBonnin

- The negation is independent of the issue. 

   The rule _for old content if the flag `hideReaderRevenue` is absent you could have be considered `true`_ is equivalent to _for old content the `showReaderRevenue ` flag would be absent and therefore could be considered `false`_

- Before the cutoff date the rule need to be based on the `sensitive` flag as it currently is.

The way to remove a cutoff date will be for editorial people to set when necessary  `hideReaderRevenue` flag on all of our old content, which they don't want to do OR accept that reader revenue may show on old content.

Hope this helps